react/features/calendar-sync/middleware.js

```diff
@@ -200,7 +200,10 @@ function _getURLFromEvent(event, knownDomains) {
                     = urlRegExp.exec(field) || schemeRegExp.exec(field))
                         !== null
             ) {
-                return matchArray[0];
+                const parsedURI = parseURIString(matchArray[0]);
+
+                // eslint-disable-next-line max-len
+                return `${parsedURI.protocol}//${parsedURI.host}/${parsedURI.room}${parsedURI.search}${parsedURI.hash}`;
```

Comment:
Please break the line for readability. These can be broken safely within the `${xxx}`. I don't remember which eslint was also OK with but it's either right after `${` or right before '}'.

Reply:
Seems to work both ways, fixed, thanks! (also fixed a similar line above that)